site/en/blog/new-in-chrome-96/index.md

```diff
@@ -0,0 +1,222 @@
+`url` that should be opened when clicked.
+
+```json
+  ...
+  "protocol_handlers": [
+    {
+      "protocol": "web+tea",
+      "url": "/tea?type=%s"
+    },
+    {
+      "protocol": "web+coffee",
+      "url": "/coffee?type=%s"
+    }
+  ]
+}
+```
+
+There are some restrictions here, and you can't just register any protocol, so
+check out [URL protocol handler registration for PWAs][wd-phreg] for complete
+details, and how you can use the `web+` syntax to create your own protocols!
+
+## Priority hints (origin trial) {: #pri-hints }
+
+When a browser parses a web page and begins to discover and download resources
+such as images, scripts, or CSS, it assigns them a fetch priority to try to
+optimize the page load. Browsers are pretty good at assigning priorities, but
+may not be optimal in all cases.
+
+Priority Hints are an experimental feature, available as an
+[origin trial][ph-ot] starting in Chrome 96, and can help optimize the Core
+Web Vitals. The `importance` attribute allows you to specify the priority
+for resource types such as CSS, fonts, scripts, images, and iframes.
+
+```html
+<!-- We don't want a high priority for this above-the-fold image -->
+<img src="/not-important.svg" importance="low">
+
+<!-- Initiate an early fetch for a resource, but de-prioritize it -->
+<link rel="preload" href="/script.js" as="script" importance="low">
+
+<script>
+  fetch('https://example.com/', {importance: 'high'}).then(data => {
+    // Trigger a high priority fetch
+  });
+</script>
+```
+
+For example, here's the Google Flights page. That background image is the
+largest contentful paint (LCP).
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/zdnbrVyWw7AMcWXVIT2g.png", alt="Screenshot of Google Flights with large background image", width="800", height="533" %}
+
+Let's see it loaded with, and without priority hints. With the priority set
+to `high` on the background image, the LCP drops from 2.6 seconds to 1.9
+seconds.
+
+{% Video src="video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/MGM6YFi3IHrjRoguRr92.mp4", autoplay="true", muted="true", loop="true" %}
+
+Check out [Optimizing resource loading with Priority Hints][wd-phints] for
+all the details, how to register for the origin trial, and some great examples
+of how it can help to improve your rendering performance.
+
+## Emulate Chrome 100 in UA string {: #chrome-100 }
+
+Early next year, we'll hit Chrome 100, a **three** digit version number. Any
+code that checks version numbers, or parses the UA string, should be checked
+to make sure it handles three digits.
+
+Starting in Chrome 96, there's a new flag `#force-major-version-to-100` that
+will change the current version number to 100, so you can make sure
+everything works as expected.
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Screenshot of Chrome Flags page highlighting new #force-major-version-to-100 option", width="800", height="533" %}
```

Comment:
```suggestion
{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Chrome Flags page highlighting new #force-major-version-to-100 option", width="800", height="533" %}
```

Reply:
Fixed in 1475b8e031abe034773b744274e64066738153ad